onnxruntime/python/tools/transformers/models/whisper/convert_to_onnx.py

```diff
@@ -378,6 +380,18 @@ def main(argv=None):
         chain_model(args)
         output_paths.append(args.beam_model_output_dir)
 
+        # Change directory layout from <output dir>/openai/<all models> to <output dir>/<model name>_beamsearch.onnx[.data]
+        # if using pre-trained model name
+        if re.search("^openai/whisper-[a-z]*", args.model_name_or_path):
+            old_model_path = os.path.join(args.output, args.model_name_or_path + "_beamsearch")
```

Comment:
This logic is tricky.  Can we just output the model to args.beam_model_output_dir?

Reply:
Currently, all models are saved in `args.beam_model_output_dir`. Using Whisper tiny as an example, because the model is named as `openai/whisper-tiny_beamsearch.onnx`, an extra `openai` folder is created within `args.beam_model_output_dir`.

Since different model sizes and variations can be exported, the regex checks for just the `openai/whisper-` portion. The directory organization is then changed from
```
args.beam_model_output_dir
├── openai
    ├── whisper-tiny_beamsearch.onnx
    ├── whisper-tiny_beamsearch.onnx.data
    ├── whisper-tiny_decoder.onnx
    ├── whisper-tiny_decoder.onnx.data
    ├── whisper-tiny_decoder_fp16.onnx
    ├── whisper-tiny_decoder_fp16.onnx.data
    ├── whisper-tiny_encoder_decoder_init.onnx
    ├── whisper-tiny_encoder_decoder_init.onnx.data
    ├── whisper-tiny_encoder_decoder_init_fp16.onnx
    ├── whisper-tiny_encoder_decoder_init_fp16.onnx.data
    └── ...
```
to
```
args.beam_model_output_dir
├── whisper-tiny_beamsearch.onnx
└── whisper-tiny_beamsearch.onnx.data
```

Reply:
A simple fix is not create openai folder (or add openai to path) at the beginning.

Reply:
The beginning of the ONNX model name is from the input to the `-m/--model` flag, which is also used as the name for getting the pre-trained PyTorch model. Since the `openai` part is needed to download the PyTorch model, I'll rewrite the logic for how the ONNX model is saved and then remove the regex.